code/modules/research/nanites/public_chamber.dm

```diff
@@ -126,7 +126,7 @@
 	if(!state_open)
 		return FALSE
 
-	..()
+	..(user)

```

Comment:
wait what

Reply:
yeah wait what does this change

Reply:
This should do nothing

Reply:
Well adding it fixed the issue, and removing it causes the issue to return, sooooo.

https://github.com/tgstation/tgstation/blob/1b50671ca072dd646e98bee971a6d20d5c06a2de/code/game/machinery/dna_scanner.dm#L92-L96

https://github.com/tgstation/tgstation/blob/cf818aaa37216dc38ca4e0d09db7c114cfac6f01/code/modules/research/nanites/nanite_chamber.dm#L189-L194

https://github.com/tgstation/tgstation/blob/7dc8c57f9e6f6e03aebb763f596245eed72751cf/code/modules/antagonists/abductor/machinery/experiment.dm#L35-L40

https://github.com/tgstation/tgstation/blob/f735a7d34dc17f7556bf109037fe660545c9abe3/code/game/machinery/Sleeper.dm#L73-L75

https://github.com/tgstation/tgstation/blob/90bda1452ba77412295cd43e341062a03bbb36af/code/modules/atmospherics/machinery/components/unary_devices/cryo.dm#L254-L256